subworkflows/local/prepare_genome/tests/main.nf.test

```diff
@@ -962,7 +948,6 @@ nextflow_workflow {
                     workflow.out.gene_bed,
                     workflow.out.chrom_sizes,
                     workflow.out.splicesites,
-                    // workflow.out.bbsplit_index,
```

Comment:
@adamrtalbot these were commented because the bbsplit index wasn't snapshottable? Should we check existence instead?

Reply:
Cool. Let's leave it out for now, and we can add them later if required.